0.download-data/README.md

```diff
@@ -0,0 +1,31 @@
+# Download CFReT data
+
+The information regarding the CFReT pilot data are as follows:
+
+```
+Assay: Cell Painting
+
+Channels: d0 (Nuclei), d1 (Endoplasmic Reticulum), d2 (Golgi/Plasma Membrane), d3 (Mitochondria), d4 (F-actin)
+
+Cells: Cardiac Fibroblasts
+
+Samples: Cells from 3 patients that suffered cardiac arrest
```

Comment:
I believe there are perturbations in this dataset as well, correct?

There is a negative control and 1 drug treatment, I believe?

Reply:
Ahh yes correct, I was getting confused with their diagram so I will talk to you in person about this to make sure that I fully understand what they are doing. I can even add in their diagram of the plate with perturbations in some part of this README as well.